pages/common/setquota.md

```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota  --remote`
+
+- Set user quotas for named user:
+
+`setquota --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
+
+- Set group quotas for named group:
+
+`setquota --group`
```

Comment:
```suggestion
`setquota --group {{value}}`
```

Optional fix: invalid command syntax: argument for `--group` is expected.
Note: addresses [this](https://github.com/tldr-pages/tldr/pull/9413/files#r1019373423) comment.